core/shared/src/main/scala/org/typelevel/log4cats/noop/NoOpLogger.scala

```diff
@@ -31,80 +31,29 @@ object NoOpLogger {
     if (evaluateArgs) strictImpl else lazyImpl
 
   private def lazyImpl[F[_]: Applicative] = new SelfAwareStructuredLogger[F] {
+    protected def kernel: LoggerKernel[F, String] = new LoggerKernel[F, String] {
+      def log(level: KernelLogLevel, record: Log.Builder[String] => Log.Builder[String]): F[Unit] =
+        Applicative[F].pure(())
+    }
```

Comment:
cleanup: use the implementation you already have
```suggestion
    protected def kernel: LoggerKernel[F, String] = NoOpLoggerKernel[F, String]
```